packages/cli/src/assets/dioxus.toml

```diff
@@ -14,16 +14,17 @@ out_dir = "dist"
 # resource (static) file folder
 asset_dir = "public"
 
+[watcher]
+
+index_on_404 = true
```

Comment:
Renaming web.watcher to watcher probably makes sense, but it could be confusing that index_on_404 doesn't effect liveview or fullstack if it isn't marked as a web property